test/e2e/performanceprofile/functests/5_latency_testing/latency_testing.go

```diff
@@ -254,34 +259,34 @@ func getNegativeTests(toolToTest string) []latencyTest {
 		latencyFailureMsg = hwlatdetectFail
 	}
 
-	testSet = append(testSet, latencyTest{testDelay: "0", testRuntime: "5", testMaxLatency: "1", outputMsgs: []string{latencyFailureMsg, fail}, toolToTest: toolToTest})
```

Comment:
adding ginkgo.timeout value to the negative sets looks risky. The goal of these tests is to test that the latency tests fail on the expected reason. If for any reason a negative test failed for timing out because of ginkgo 10s timeout, this will not give us any clue on whether the check passed or not. 
implementing a negative test for the original bug should be written a bit differently than the positive. for now, what I'd go with is running negative tests with default ginkgo.timeout. let me know what you think.

Reply:
Thanks for the feedback and apologies for the delay in getting back to this. I think your suggestion makes sense. My intention was to keep the ginkgo timeout high enough compared to the runtime to not interfere with the original intent of the test I will update now so that the negative tests are executed with the default ginkgo timeout.